style.css

```diff
@@ -0,0 +1,273 @@
+:root {
+    --primary-main: #0067fc;
+    --primary-light: #f0f8ff;
+    --primary-dark: #0048cd;
+    --white-main: #fff;
+
+    --font-size-extra-small: 0.75rem;
+    --font-size-small: 0.875rem; 
+    --font-size-medium: 1rem; 
+    --font-size-large: 1.5rem;
+
+    --size-1: 0.25rem;
+    --size-2: 0.5rem; 
+    --size-3: 1rem; 
+    --size-4: 2rem; 
+
+    --transition-duration: 0.3s;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+.container {
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    position: sticky;
+    height: 65px;
+    top: 0;
+    left: 0;
+    background-color: var(--white-main);
+    padding-right: var(--size-2);
+}
```

Comment:
As a general rule, you can put yourself in a corner by styling html elements by themselves. In such case you couldn't use header element anywhere else -- and it is possible to have multiple headers in a page. Read more on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header#article_header 

I advise to move these styles to a class `.header` for example :) So that:

```
.header
    .container.flex-between
        ...
```